src/tr1/game/text.c

```diff
@@ -200,8 +200,9 @@ void Text_DrawText(TEXTSTRING *const text)
             goto loop_end;
         }
         Output_DrawScreenSprite2D(
-            sx, sy, 0, sh, sv, g_Objects[O_ALPHABET].mesh_idx + glyph->mesh_idx,
-            16 << 8, 0, 0);
+            sx, sy, 0, sh, sv,
+            Object_GetObject(O_ALPHABET)->mesh_idx + glyph->mesh_idx, 16 << 8,
```

Comment:
This can be just `obj`.

Reply:
BTW it'd be good to settle on whether we prefer `obj` or `object`. I'm fine with either.

Reply:
Thanks, yes I think we should go with `obj` as we currently have more of that than `object`. I'll follow up with a separate PR as the renames will be more involved.